### PR TITLE
Create Plugins.java

### PR DESCRIPTION
**Pull request by hoyinm**

### CODE REVIEW

**Comment by PEMapModder** [comment]
Do you know Java... Class names and file names..

**Comment by PhilipShilling** [comment]
@PEMapModder It appears that he was simply copying the Help command to make the plugins command

**Comment by PEMapModder** [comment]
It appears that he doesn't know Java.

File: src/redstonelamp/cmd/commands/Plugins.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	public void onServerStart(ServerStartEvent event) {
+		RedstoneLamp.server.getCommandRegistrationManager().registerCommand("plugins", "Shows a list of all plugins", true);
+	}
+	
+	public boolean onCommand(CommandSender sender, Command cmd, String label, List<String> args) {
+		switch(cmd.getName()) {
+			case "plugins":
+			case "pl":
+				if(sender->isOp()){
+				  //TODO: Send list of plugins
```
Comment (PEMapModder) [inline src/redstonelamp/cmd/commands/Plugins.java]: Well... What's the point of this then?

Comment (PhilipShilling) [inline src/redstonelamp/cmd/commands/Plugins.java]: @PEMapModder +1

**Comment by EnderBrine101** [comment]
Lol #Rekt

**Pull request closed without merging**